src/com/javabasic/lesson14oop/HomeWorkProcessor.java

```diff
@@ -0,0 +1,39 @@
+package com.javabasic.lesson14oop;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class HomeWorkProcessor {
+
+    public static void main(String[] args) throws InterruptedException, IOException {
+
+        // Task №1
+        List<RealEstateTax> taxes = new ArrayList<>();
```

Comment:
I suggest renaming this variable to estates and removing Tax postfix from class names. As we create objects of Real Estate and calculate their tax. Another sign is that area property is related to real estate objects.

Reply:
Done